src/test/java/com/example/artvswar/dto/mapper/AccountMapperTest.java

```diff
@@ -0,0 +1,233 @@
+package com.example.artvswar.dto.mapper;
+
+import com.example.artvswar.dto.request.account.AccountCreateUpdateRequestDto;
+import com.example.artvswar.dto.request.account.AccountShippingRequestDto;
+import com.example.artvswar.dto.response.account.AccountResponseDto;
+import com.example.artvswar.dto.response.account.AccountShippingResponseDto;
+import com.example.artvswar.model.Account;
+import com.example.artvswar.model.AccountEmailData;
+import com.example.artvswar.model.AccountShippingAddress;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+@Order(300)
+@SpringBootTest
```

Comment:
Here we agreed to delete @SpringBootTest and make this test only with mock as we do not need full context to test logic here